docs/user/intelmq-api.rst

```diff
@@ -0,0 +1,112 @@
+
+``intelmq-api`` is configured using a configuration file in ``json`` format.
+The path to the configuration file is set using the environment variable ``INTELMQ_API_CONFIG``.
+When running the API using ``hug``, you can set the environment variable like this:
+
+.. code-block:: bash
+
+   INTELMQ_API_CONFIG=/etc/intelmq-api/config.json hug -m intelmq_api.serve
+
+
+The configuration file ``/etc/intelmq-api/config.json`` which is shipped with the packages is also listed here fore reference.
+This also gives an example on how to disable a setting, namely by prefixing the name with an underscore, like it is done here with the ``_session_store`` setting.
+
+.. code-block:: json
+
+   {
+           "intelmq_ctl_cmd": ["intelmqctl"],
+           "allowed_path": "/opt/intelmq/var/lib/bots/",
+           "_session_store": "/tmp/intelmq-session.sqlite",
+           "session_duration": 86400,
+           "allow_origins": ["*"],
+           "html_dir": "/usr/share/intelmq-manager/html/"
+   }
+
+The following configuration options are available:
+
+* ``intelmq_ctl_cmd``: Your ``intelmqctl`` command. If this is not set in a configuration file the default is used, which is ``["sudo", "-u", "intelmq", "/usr/local/bin/intelmqctl"]``
+  The option ``"intelmq_ctl_cmd"`` is a list of strings so that we can avoid shell-injection vulnerabilities because no shell is involved when running the command.
+  This means that if the command you want to use needs parameters, they have to be separate strings.
+* ``allowed_path``: intelmq-api can grant **read-only** access to specific files - this setting defines the path those files can reside in.
+* ``session_store``: this is an optional path to a sqlite database, which is used for session storage and authentication. If it is not set (which is the default), no authentication is used!
+* ``session_duration``: the maximal duration of a session, its 86400 seconds by default
+* ``allow_origins``: a list of origins the responses of the API can be shared with. Allows every origin by default.
+* ``html_dir``: the path to the html files of the ``intelmq-manager``. If this path exists it is served under the path ``/management``
+
+*************
+Adding a user
+*************
+
+If you enable the ``session_store`` you will have to create user accounts to be able to access the API functionality. You can also do this using hug:
+
+.. code-block:: bash
+
+   hug -m intelmq_api.serve -c add_user <username>
+
+**************
+Usual problems
+**************
+
+If the command is not configured correctly, you'll see exceptions on startup like this:
+
+.. code-block:: bash
+
+   intelmq_manager.runctl.IntelMQCtlError: <ERROR_MESSAGE>
+
+This means the intelmqctl command could not be executed as a subprocess.
+The ``<ERROR_MESSAGE>`` should indicate why.
+
+To save the positions of the bots in the configuration map, you need an existing writable ``manager/positions.conf`` file. If it's missing, just create an empty one.
```

Comment:
... and set the owner?

Reply:
I've now replaced that sentence with a paragraph about permissions in the configuration section.